tests/integration/compose/docker_compose_hdfs.yml

```diff
@@ -1,6 +1,6 @@
 services:
     hdfs1:
-        image: prasanthj/docker-hadoop:2.6.0
+        image: gvacaliuc/hadoop-docker:2.7.4
```

Comment:
HI @HowePa ,

Just curious if there are a lot more involved to use the official image repository from apache - https://hub.docker.com/r/apache/hadoop/tags ? 

Although it is part of test suite and i checked the image and it is fine, I definitely wont recommend anyone using this in their environment. docker scout returns the following list of vulnerabilities:

571 vulnerabilities found in 80 packages
  LOW       98
  MEDIUM    228
  HIGH      188
  CRITICAL  57